docker/serverless/env-start-esp.py

```diff
@@ -59,9 +59,10 @@ def serve_error_msg(error_msg):
 
 def main():
     CMD = "/usr/sbin/start_esp"
+    PLATFORM = "Cloud Run"
 
     # The command being run must be the 0th arg.
-    ARGS = [CMD, "--enable_backend_routing"]
+    ARGS = [CMD, "--enable_backend_routing", "--compute_platform_override={}".format(PLATFORM)]
```

Comment:
I'm not 100% sure how spaces would be handled for an arg that's effectively `"--compute_platform_override=Cloud Run"`; have you tested this?

Reply:
Oops, thanks for catching it!

We don't currently have a test for this start up script unfortunately. I believe there is a plan to add an e2e test.